content/en/blog/_posts/2023-03-06-node-log-query-alpha.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Query Node Logs Using The Kubelet API"
+date: 2023-06-03
```

Comment:
```suggestion
date: 2023-04-21
```

Reply:
Please also change the path (to the Markdown file) to match this date.

Reply:
Done